lib/OpenQA/Worker/Commands.pm

```diff
@@ -148,19 +145,20 @@ sub websocket_commands {
             if ($job && $job->{id}) {
                 $OpenQA::Worker::Common::job = $job;
                 log_debug("Job " . $job->{id} . " scheduled for next cycle");
+                my $send_status = sub {
+                    log_debug("Sending IMMEDIATELY worker status to $host");
```

Comment:
I saw that you just moved this debug line but I wanted to ask you anyway what it means. I don't understand the english sentence here. Could you clarify that line?